scripts/delete-inactive-helm-releases.sh

```diff
@@ -2,7 +2,6 @@
 azureResourceGroup=${1:cft-preview-00-rg}
 kubernetesCluster=${2:cft-preview-00-aks}
 defaultInactiveDays=${3:-3}
-declare -A inactiveDaysOverride=(["civil"]=1) # Value should be less than defaultInactiveDays defined above
```

Comment:
You do not need to revert the functionality. Just remove civil from this map. This might still be needed if we see any issues for a namespace in future.

Reply:
You mean to leave it empty ?
`
declare -A inactiveDaysOverride=(["civil"]=1)`